rpcd/playbooks/roles/rpc_maas/tasks/local.yml

```diff
@@ -195,6 +195,21 @@
 
 - include: local_setup.yml
   vars:
+    check_name: neutron_metadata_agent_proxy_check
+    check_details: file=neutron_metadata_local_check.py,args={{ internal_vip_address }}
+    check_period: "{{ maas_check_period }}"
+    check_timeout: "{{ maas_check_timeout }}"
+    alarms:
+      - { 'name': 'neutron_metadata_agent_proxy_status', 'criteria': ':set consecutiveCount={{ maas_alarm_local_consecutive_count }} if (metric["neutron-metadata-agent-proxy_status"] != 1) { return new AlarmStatus(CRITICAL, "neutron-metadata-agent-proxy non-responsive"); }' }
```

Comment:
Hi @stevelle , metric["neutron-metadata-agent-proxy_status"] doesn't exist, the plugin is writing 'neutron_metadata_agent_proxies_on_host_%s' % container.  I suppose you'd need to set up an alarm for each container returned by the check?

On a side note, I'm not sure any of the existing checks/alarms support running multiple containers for a service on a single host.  This isn't ideal but just something to think about as you implement this.